sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/MessageSerializer.java

```diff
@@ -0,0 +1,37 @@
+package com.azure.messaging.eventhubs.implementation;
+
+import org.apache.qpid.proton.message.Message;
+
+/**
+ * Serializer to translate objects to and from proton-j {@link Message messages}.
+ */
+public interface MessageSerializer {
```

Comment:
Would this be a candidate for core-amqp module instead of Event Hub module?

Reply:
Yup! This will be moved there in the PR https://github.com/Azure/azure-sdk-for-java/pull/5346

Reply:
Actually. I'll move it there now.